db/migrate/20170210085327_create_boards.rb

```diff
@@ -0,0 +1,10 @@
+class CreateBoards < ActiveRecord::Migration[5.1]
+  def change
+    create_table :boards do |t|
+      t.string :name
+      t.references :node, foreign_key: { on_delete: :cascade }
```

Comment:
Instead of adding this in  a later migration like we had to do in Pro, here we can add this reference when creating boards.
This line replaces: `t.integer :project_id, index: true`